server/node/src/main/java/org/kaaproject/kaa/server/control/service/DefaultControlService.java

```diff
@@ -1124,14 +1111,22 @@ public FileData generateSdk(SdkProfileDto sdkProfile, SdkPlatform platform) thro
             }
         }
 
+
         LOG.debug("Sdk profile for sdk generation: {}", sdkProfile);
 
         SdkGenerator generator = SdkGeneratorFactory.createSdkGenerator(platform);
+
+        boolean isTrustfulCredentialService = isTrustfulCredentialService(sdkProfile.getApplicationId());
+        if (!(generator instanceof JavaSdkGenerator)) {
+            isTrustfulCredentialService = true; // only for java, APP-1190
```

Comment:
Do not leave such comments in code.
We have some commit messages format:
"<JIRA_TIKET_ID>: commit message"
If someone will have some issues with your code he will find all necessary information from ticket using ticket_id from commit message.